base_layer/contacts/examples/chat_client/src/networking.rs

```diff
@@ -22,6 +22,8 @@
 
```

Comment:
Maybe not related to this PR:

This function
`\base_layer\contacts\examples\chat_client\src\networking.rs` :: `pub async fn start`
is called from
`\base_layer\contacts\examples\chat_client\src\client.rs` :: `pub async fn initialize`
is called from
`base_layer\chat_ffi\src\lib.rs` :: `pub unsafe extern "C" fn create_chat_client`

Why are those functions in an example if they appear to be library code?

Reply:
I've been waiting for someone to ask!

This is basically a refactoring waiting to happen. Initially, this client was an example and evolved into a library. It should absolutely be moved elsewhere. Did you have an opinion on where? From the example lib out one directory into contacts makes sense.

Reply:
Why dont we just rename the path to remove example, and if we have actual example only code, we can move that to `\base_layer\contacts\chat_client\examples\src\xx.rs`

Reply:
Yes, that could work.